<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>452</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="theme_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="theme_view">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>341</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="package_button">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>310</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Package</string>
    </property>
   </widget>
   <widget class="QPushButton" name="revert_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Revert</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
